README.md

```diff
@@ -41,6 +41,19 @@ steps:
 - run: npm test
 ```
 
+Checking latest version of Node.js:  
```

Comment:
I think we should add a note about it being a tradeoff between perf / reliability and ensuring latest.  Default of check-latest false ensure instant resolution to major / minor version with up to a couple of weeks latency.  Check-latest true reduces latency significantly but is much more likely to incur a download and extract.   Saying that will help users make the trade-off.  I want to make sure they think about how badly they want that reduced latency.

Reply:
I have added a few more sentences. Please let me know if wording should be changed.